addons/web/static/src/views/calendar/calendar_model.js

```diff
@@ -533,6 +529,7 @@ export class CalendarModel extends Model {
             endType,
             duration,
             colorIndex,
+            allday: rawRecord["allday"] || false,
```

Comment:
Use `rawRecord.allday` directly in the template `calendar.AttendeeCalendarCommonRenderer.event` instead.